### PR TITLE
create dummy all tests script for openshift-ci 

### PR DESCRIPTION
**Pull request by wtam2018**
**What type of PR is this?**
/enhancement

**What does this PR do / why we need it**:
We need a script to be called by openshift-ci to invoke in order the kick off e2e tests.
See https://github.com/openshift/release/pull/12283

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes NA

**How to test changes / Special notes to the reviewer**:
pr https://github.com/openshift/release/pull/12283 should pass

### CODE REVIEW

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+ARCH=$(uname -m)
+export CI="openshift"
+if [ "${ARCH}" == "s390x" ]; then
+    make configure-installer-tests-cluster-s390x
+elif [ "${ARCH}" == "ppc64le" ]; then
+    make configure-installer-tests-cluster-ppc64le
+else
+    make configure-installer-tests-cluster
+fi
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:16]: Remove line 8 to lien 16. These are not useful atm

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+make bin
+mkdir -p $GOPATH/bin
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:18]: Please remove it. Not useful atm

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+go get -u github.com/onsi/ginkgo/ginkgo
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:19]: Please remove it. Not useful atm

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+export PATH="$PATH:$(pwd):$GOPATH/bin"
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:20]: ```export PATH="$PATH:$(pwd):$GOPATH/bin"``` -> ```export PATH="$PATH:$(pwd)"```

Adding pwd in the path will help us to detect the `kam` bit

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+# Copy kubeconfig to temporary kubeconfig file
+# Read and Write permission to temporary kubeconfig file
+TMP_DIR=$(mktemp -d)
+cp $KUBECONFIG $TMP_DIR/kubeconfig
+chmod 640 $TMP_DIR/kubeconfig
+export KUBECONFIG=$TMP_DIR/kubeconfig
+
+# Login as developer
+odo login -u developer -p developer
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:32]: Remove line 24 to line 32. Prow by default provide the kubeadmin login instance. Later we will look for htpasswd login if needed, So these are not useful atm

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+# Check login user name for debugging purpose
+oc whoami
+
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:36]: Please add ```kam version``` just to check kam is build properly and version command is executed. Later we will jam in more information.

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    #make test-e2e-beta
+else
+    # Integration tests
+    # make test-integration
+
+    # E2e tests
+    #make test-e2e-all
+fi
+
+odo logout
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:57]: Remove line 37 to line 57. These are not useful atm.

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: scripts/openshiftci-presubmit-all-tests.sh (hunk 4 above)
Comment (wtam2018) [reply scripts/openshiftci-presubmit-all-tests.sh:20]: ${pwd} is already added there ^^^, right?

Comment (amitkrout) [reply scripts/openshiftci-presubmit-all-tests.sh]: yes

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 8]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+make bin
+export PATH="$PATH:$(pwd)"
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:9]: +1

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    # E2e tests
+elif  [ "${ARCH}" == "ppc64le" ]; then
+    # Integration tests
+
+    # E2e tests
+else
+    # Integration tests
+
+    # E2e tests
+fi
```
Comment (amitkrout) [inline scripts/openshiftci-presubmit-all-tests.sh:27]: Please remove from line 15 to line 27. These are the platform supported condition and till now ```kam``` does not release binary for such distribution of linux platform.

**Review by amitkrout — CHANGES_REQUESTED** [review]

**Pull request merged**